simphony/cuds/lattice.py

```diff
@@ -132,6 +137,34 @@ def get_coordinate(self, index):
         else:
             return self.origin + self.base_vect*np.array(index)
 
+    def count_of(self, item_type):
+        """ Return the count of item_type in the container.
+
+        Parameter
+        ---------
+        item_type : CUBA
+            The CUBA enum of the type of the items to return the count of.
+
+        Returns
+        -------
+        count : int
+            The number of items of item_type in the container.
+
+        Raises
+        ------
+        ValueError :
+            If the type of the item is not supported in the current
+            container.
+
+        """
+
+        if item_type in self._allowed_item_types:
+            if item_type == CUBA.NODE:
+                return reduce(lambda x, y: x*y, self._size)
```

Comment:
@roigcarlo, you could replace this by
´´´python
    return np.prod(self._size)
´´´
because numpy is already imported.